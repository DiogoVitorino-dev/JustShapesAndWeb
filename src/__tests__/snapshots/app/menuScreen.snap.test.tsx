import { render } from "test-utils";

import Menu from "@/app";

describe("Menu - Snapshot app screen test", () => {
  it("should render correctly", () => {
    const tree = render(<Menu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
