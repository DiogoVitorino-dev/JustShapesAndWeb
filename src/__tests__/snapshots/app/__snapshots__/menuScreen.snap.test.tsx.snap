// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu - Snapshot app screen test should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
